akd_core/src/hash/tests.rs

```diff
@@ -112,7 +112,7 @@ fn test_merge_int_validity() {
     let random_hash = random_hash();
     let merged = merge_with_int(random_hash, random_epoch);
 
-    let data = vec![random_hash.to_vec(), random_epoch.to_le_bytes().to_vec()].concat();
+    let data = vec![random_hash.to_vec(), random_epoch.to_be_bytes().to_vec()].concat();
```

Comment:
I can't do a find-and-replace on your branch, but you're sure you got all the `le_bytes` instances? Just want to make sure there's nothing dangling left!

Reply:
I believe so:
```
klewi@klewi-mbp akd % grep -rn 'le_bytes' */src
akd_core/src/utils.rs:75:    let stale_bytes = if stale { &[0u8] } else { &[1u8] };
akd_core/src/utils.rs:79:            stale_bytes,
akd_mysql/src/mysql_storables.rs:639:                (Some(possible_bytes), Some(possible_u32)) => {
akd_mysql/src/mysql_storables.rs:642:                        from_value_opt::<Vec<u8>>(possible_bytes),
```